### DISCUSSION TITLE
[2기-A] 김현우

### DISCUSSION THREAD

**Discussion by hook0318** [super-corini/programmers_kdt_II#353] [2기 Monthly Project I]
### site : https://priceless-roentgen-58af0e.netlify.app/

AWS 배포를 시도하려 했으나 정지해놓은 줄 알았던 인스턴스는 켜져 있어서 어느새 사용량  98.9%가 되어 HTML 파일만으로 netlify를 이용한 배포를 진행했습니다. (여러분도 꺼진 AWS도 다시 한번 확인하세요.)

- ipynb -> html 변환법
`$ jupyter nbconvert --to html <변환할 ipynb파일>`

Django를 통한 AWS 배포는 사용량이 리셋되면 다시 시도해볼 생각입니다. 근데 이제 CSS와 JS를 곁들인...

감사합니다.

**Comment by super-corini** [comment]
어... 최강록 선생님의 향기가 진하네요 😎 검은색과 노란색의 Plot 대비가 인상깊습니다. 근데 이제 범블비를 곁들인...
